frontend/src/App.js

```diff
@@ -108,18 +107,21 @@ function App() {
   const [extraPixelsUsed, setExtraPixelsUsed] = useState(0);
   const [extraPixelsData, setExtraPixelsData] = useState([]);
 
+  const [isEraserMode, setIsEraserMode] = React.useState(false);
+
   useEffect(() => {
     const address = 0;
-    async function fetchExtraPixelsEndpoint() {
-      let getExtraPixelsEndpoint = await fetchWrapper(
-        `get-extra-pixels?address=${address}`
-      );
-      if (!getExtraPixelsEndpoint.data) {
-        return;
-      }
-      setExtraPixels(getExtraPixelsEndpoint.data);
-    }
-    fetchExtraPixelsEndpoint();
+    let getExtraPixelsEndpoint = `${backendUrl}/get-extra-pixels?address=${address}`;
+    fetch(getExtraPixelsEndpoint).then((response) => {
+      response
+        .json()
+        .then((result) => {
+          setExtraPixels(result.data);
+        })
+        .catch((error) => {
+          console.error(error);
+        });
+    });
```

Comment:
This might be some artifact from merging changes, but the deleted implementation is the prefered one here.